src/cloudai/systems/slurm/strategy/slurm_command_gen_strategy.py

```diff
@@ -237,3 +322,20 @@ def _append_sbatch_directives(
         batch_script_content.append(
             "\nexport SLURM_JOB_MASTER_NODE=$(scontrol show hostname $SLURM_JOB_NODELIST | head -n 1)"
         )
+
+    def _format_env_vars(self, env_vars: Dict[str, Any]) -> str:
+        """
+        Format environment variables for inclusion in a batch script.
+
+        Args:
+            env_vars (Dict[str, Any]): Environment variables to format.
+
+        Returns:
+            str: A string representation of the formatted environment variables.
+        """
+        formatted_vars = []
+        for key in sorted(env_vars.keys()):
+            value = env_vars[key]
+            formatted_value = str(value["default"]) if isinstance(value, dict) and "default" in value else str(value)
```

Comment:
We really need to clean it up, I don't think we can have a dict here. Have you seen cases where it is a dictionary and not just a string?

Reply:
I agree, but let's discuss it in a separate PR or topic. We can't address every issue in this one.